packages/messenger-widget/src/adapters/contacts.ts

```diff
@@ -215,14 +215,22 @@ export async function getContacts(
                     : normalizeEnsName(ensNames[0]),
             )
             .map(async (ensName) => {
-                const profile = await getUserProfile(
-                    connection.provider!,
-                    ensName,
-                );
-                return {
-                    ensName,
-                    profile: profile,
-                };
+                let profile;
+                try {
+                    profile = await getUserProfile(
+                        connection.provider!,
+                        ensName,
+                    );
+                    return {
+                        ensName,
+                        profile: profile,
+                    };
+                } catch (error) {
```

Comment:
Can the code proceed with the profile being undefined ?  If not might it make more sense to handle the error instead of just returning undefined

Reply:
Yes, it can proceed. When we add new contacts they might have not configured the profile, but will have ENS name and address that we need to show in contact list. Although chatting won't be allowed with those users until they configure profile, but that will be a check in chatting screen. But here simply it returns undefined showcasing that user has not configured profile.